bridge/cxx/include/bhxx/BhArray.hpp

```diff
@@ -160,9 +154,17 @@ class BhArray {
         view.base  = base.get();
         view.start = static_cast<int64_t>(offset);
         view.ndim  = static_cast<int64_t>(shape.size());
-        view.slide = slide;
-        view.slide_dim_stride = slide_dim_stride;
-        view.slide_dim_shape = slide_dim_shape;
+
```

Comment:
Create a copy constructor to `BhSlide` or if every member has a default constructor, which I think they have in this case, use `view.slides = slides`